data/json/monsters.json

```diff
@@ -992,7 +992,7 @@
     "id": "mon_zombie_flamer",
     "type": "MONSTER",
     "name": "zombie burner",
-    "description": "Realizing that bullets weren't enough, the army dispatched flamethrower units to burn the walking dead.  Unfortunately, there were too many zombies and not enough flamers.  It seems that the tank of the undead soldier you see before you is damaged.",
+    "description": "When bullets weren't enough to stop the end of the world, the military tried fire instead.  Their failure is made manifest in this gas-masked husk, its black suit torn and ripped.  Its flamethrower, dangling limply at its side, is attached to a tank of napalm strapped onto its back, which trickles its contents onto the ground.",
```

Comment:
Suggest `enough to stop the end of the world, the military tried fire instead.` -> `enough to prevent the end of the world, the military used [conflagration] instead.`

`this gas-masked husk, its black suit torn and ripped.` -> `this gas-masked husk with its black suit torn and ripped.`